certif/app/templates/components/routes/authenticated/sessions/details/certification-candidates-tab.hbs

```diff
@@ -25,61 +25,110 @@
+          <div class="panel-actions__title">Importer le PV de session</div>
+          <div class="panel-actions__subtitle">
+            Sélectionnez le PV de session préalablement rempli.
+            <br>
+            <strong>
+              Attention, tout nouvel import efface la liste des candidats existante.
+            </strong>
           </div>
-        {{else}}
-          <div class="panel-actions__button">
+        </div>
+        <div class="panel-actions__button">
+          <FileUpload @name="file-upload" @for="upload-attendance-sheet" @accept=".ods" @multiple={{false}} @onfileadd={{action importCertificationCandidates}}>
             <span data-test-id="attendance_sheet_upload_button"
-                  class="button button--with-icon button--disabled"
+                  class="button button--with-icon"
                   role="button"
-                  tabindex=0>Importer (.ods){{fa-icon 'cloud-upload-alt'}}</span>
-          </div>
-        {{/if}}
+                  tabindex=0>
+              Importer (.ods){{fa-icon 'cloud-upload-alt'}}
+            </span>
+          </FileUpload>
+        </div>
+      {{else}}
+        <div class="panel-actions__action-icon">
+          {{fa-icon 'exclamation-circle' class="panel-actions__warning-icon"}}
+        </div>
+        <div class="panel-actions__description">
+          <strong class="panel-actions__warning">
+            La session a débuté, vous ne pouvez plus importer une liste de candidats.<br>Si vous souhaitez modifier la liste, vous pouvez ajouter un candidat directement dans le tableau ci-dessous.
+          </strong>
+        </div>
+      {{/if}}
     </div>
   </div>
 </div>
 
 <div class="panel">
-  <div class="table content-text content-text--small">
-    {{#if certificationCandidates}}
+  <div class="panel-header">
+    <div class="panel-header__title">
+      Liste des candidats ({{certificationCandidates.length}})
+    </div>
+    <div data-test-id="add-certification-candidate-staging__button" class="panel-header__action" onclick={{action "addCertificationCandidateInStaging"}}>
+      <div class="certification-candidates-add-button__text">
+        Ajouter un candidat
+      </div>
+      <button class="button button--square button--lightgrey">
+        {{fa-icon "plus" class="fa-fw certification-candidates-add-button__icon"}}
+      </button>
+    </div>
+  </div>
+  <div class="table content-text content-text--small certification-candidates-table">
+    {{#if (or certificationCandidates this.candidatesInStaging)}}
       <table>
         <thead>
           <tr>
-            <th>Nom de naissance</th>
-            <th>Prénom</th>
-            <th>Date de naissance</th>
-            <th>Commune de naissance</th>
-            <th>Département de naissance</th>
-            <th>Pays de naissance</th>
+            <th class="certification-candidates-table__column-name">
+              Nom de naissance
+              {{#if this.candidatesInStaging.length}}
```

Comment:
Cela pourrait être plus explicite d'ajouter une variable spécifiant de manière plus claire pourquoi ce if est utile. Au lieu de `this.candidatesInStaging.length`, cela pourrait être `isInputMode` par exemple

Reply:
Done dans prochain push